site/about/styles.css

```diff
@@ -301,10 +242,9 @@
      display: flex;
      flex-direction: row;
      margin-top: 30px;
-     margin-left: 50px;
  }
 
- .approach-image {
+ .approach-image img {
```

Comment:
I think you should probably also set `object-fit: cover;` on this `img` as well. If you set `height` and `width` the image can get squished/squeezed. The `object-fit` will make it so the image does not squish/squeeze.